common/src/main/java/com/microsoft/identity/common/internal/controllers/ExceptionAdapter.java

```diff
@@ -188,24 +201,70 @@ public static ServiceException getExceptionFromTokenErrorResponse(@NonNull final
         }
 
         outErr.setOauthSubErrorCode(errorResponse.getSubError());
+        setHttpResponseUsingTokenErrorResponse(outErr, errorResponse);
+        return outErr;
+    }
+
+
+    public static ServiceException getExceptionFromTokenErrorResponse(@Nullable final CommandParameters commandParameters,
+                                                                      @NonNull final TokenErrorResponse errorResponse) {
+
+        if(isIntunePolicyRequiredError(errorResponse)){
+            if(commandParameters == null || !(isBrokerTokenCommandParameters(commandParameters))){
+                Logger.warn(TAG, "In order to properly construct the IntuneAppProtectionPolicyRequiredException we need the command parameters to be supplied.  Returning as service exception instead.");
+                return getExceptionFromTokenErrorResponse(errorResponse);
+            }
+            IntuneAppProtectionPolicyRequiredException policyRequiredException;
+            if(commandParameters instanceof BrokerInteractiveTokenCommandParameters) {
```

Comment:
```suggestion
            if (commandParameters instanceof BrokerInteractiveTokenCommandParameters) {
```

Reply:
I ran the code formatter in Android Studio.